tests/infra/ccf.py

```diff
@@ -5,12 +5,12 @@
 import logging
 from contextlib import contextmanager
 from enum import Enum, IntEnum
-import infra.clients
+from ccf.clients import CCFConnectionException
```

Comment:
Nit: To avoid confusion between `ccf` and `infra.ccf`, perhaps this should be `infra/network.py`? This just means more renames, and I think we moved from `network.py` to `ccf.py` already...

Reply:
I did actually mean to do that at some point but never got around to doing. I guess I can't postpone this anymore... ;)

Reply:
Done.